framework/web/Request.php

```diff
@@ -1477,6 +1493,181 @@ protected function loadCookies()
         return $cookies;
     }
 
+    /**
+     * Returns uploaded files for this request.
+     * Uploaded files are returned in format according to [PSR-7 Uploaded Files specs](http://www.php-fig.org/psr/psr-7/#16-uploaded-files).
+     * @return array uploaded files.
```

Comment:
Isn't the return type actually more specific`Psr\Http\Message\UploadedFileInterface[]`?

Reply:
No array can be multidimentional with unknown depth, e.g.:

```php
[
    'Item' => [
        'file' => [
            0 => UploadedFileInterface()
        ],
    ],
]
```